module-apps/application-call/data/CallAppStyle.hpp

```diff
@@ -84,6 +84,13 @@ namespace callAppStyle
             inline constexpr auto w = 240U;
             inline constexpr auto h = 20U;
         } // namespace durationLabel
+        namespace providedNumbersDuringTheCallLabel
```

Comment:
I'd avoid creating such long names with, especially with particles in them. Maybe something more like `enteredDigitsField` or `enteredDigitsLabel`?